pkg/api/v20240812preview/openshiftcluster.go

```diff
@@ -276,6 +288,33 @@ type IngressProfile struct {
 	IP string `json:"ip,omitempty" swagger:"readOnly"`
 }
 
+type PlatformWorkloadIdentityProfile struct {
+	PlatformWorkloadIdentities []PlatformWorkloadIdentity `json:"platformWorkloadIdentities,omitempty"`
+}
+
+// PlatformWorkloadIdentity stores information representing a single workload identity.
+type PlatformWorkloadIdentity struct {
+	Name       string `json:"name,omitempty"`
```

Comment:
This was originally `Name`, but we later changed it to `OperatorName` in the design doc to make the purpose of the field more clear.